lib/taurus/core/tango/tangovalidator.py

```diff
@@ -90,19 +90,22 @@ def getNames(self, fullname, factory=None, queryAuth=True):
         if groups is None:
             return None
 
-        import PyTango
-        default_authority = '//' + PyTango.ApiUtil.get_env_var('TANGO_HOST')
+        default_authority = None
```

Comment:
This is breaking the API that says that the authority is not queried unless `queryAuth` is True